docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/_index.md

```diff
@@ -4,8 +4,6 @@ headerTitle: xCluster Disaster Recovery
 linkTitle: Disaster recovery
 description: Enable deployment using transactional (active-standby) replication between universes
 headContent: Fail over to a replica universe in case of unplanned outages
-tags:
-  feature: early-access
 menu:
```

Comment:
see also https://docs.yugabyte.com/preview/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup/#prerequisites

Reply:
Actually, I will delete this in the DR scoped PR